.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -61,6 +63,54 @@ object WPComPlugins : Project({
 	}
 })
 
+object BuildPlugins: BuildType({
+	id("calypso_WPComPlugins_Build_Plugins")
+	uuid = "8453b8fe-226f-4e91-b5cc-8bdad15e0814"
+	name = "Build WPCom Plugins"
+	description = "Test"
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	artifactRules = """
+		apps/happy-blocks/release-files => happy-blocks.zip
+		apps/notifications/dist => notifications.zip
+	""".trimIndent()
+
+	steps {
+		mergeTrunk()
+
+		bashNodeScript {
+			name = "Prepare environment"
+			scriptContent = """
+				set -x
+
+				# Update composer
+				composer install
+
+				# Install dependencies
+				yarn
+
+				# Set execution permission for additional scripts.
+				chmod +x .teamcity/scripts/WPComPlugins/
+			"""
+		}
+
+		bashNodeScript {
+			name = "Build artifacts"
+			scriptContent = """
+				# @automattic/wpcom-block-editor needs this variable.
+				export NODE_ENV="development"
+
+				# Run `yarn build` for the plugins specified in the glob.
+				yarn workspaces foreach --verbose --parallel --include '{happy-blocks,@automattic/notifications}' run build-ci
```

Comment:
@noahtallen moving the conversation about outputs here.

As I noted in [this mainline comment](https://github.com/Automattic/wp-calypso/pull/72934#issuecomment-1433730106) the `yarn workspaces foreach` command appears to batch together outputs as the task completes one by one.

In the comment was an example output for 3 builds - `happy-blocks` which is the fastest, followed by `@automattic/notification` and `@automattic/inline-help`.

I'm typically in favor of using whatever ability the tools have natively, but understand that using `runTask` would "standardize" the behavior. What do you think?

Reply:
I think it depends what else we have to do to get it working. I think if JS would end up being a more friendly language for bootstrapping and bringing everything together, `runTask` makes sense. But if it's as simple as a one-line bash command, keeping it as bash makes sense

Reply:
> I think it depends what else we have to do to get it working. I think if JS would end up being a more friendly language for bootstrapping and bringing everything together, `runTask` makes sense. But if it's as simple as a one-line bash command, keeping it as bash makes sense

From what I see so far, it would be a one-line bash call **unless** we start adding much more post-processing. At that point, it will make sense to switch over to using `runTask` and build the commands there.